Fandom Creator/JavaScript.md

```diff
@@ -34,6 +36,30 @@ class Toggle extends React.Component {
 }
 ```
 
+## Instance fields in a class
+Similar to the public class instance methods, instance fields do not need to be set in the constructor. All instance fields should be initialized in the root of the class. Fields should _always_ be declared as "instance fields" over setting them in the constructor (even when a constructor is required for other reasons). 
```

Comment:
I think the guideline text "MUST" include the MUST from above. All I see here is "should" and "always".